audio/writer-lame.h

```diff
@@ -0,0 +1,12 @@
+#ifndef AUDIO_WRITER_LAME_H
+#define AUDIO_WRITER_LAME_H
+
+#include "writer.h"
+
+#ifdef WITH_LAME
+struct audio_writer *audio_writer_lame(FILE *outfile, int sample_rate, int nchannels, int buf_time_s, int mono);
+#else
+#define audio_writer_lame(...) (NULL)
```

Comment:
I think this illustrates that these should probably be modular and probably loaded with dlopen(3) either automatically or based on a configuration. I have some idea how I'd like that to look if you want to do that later, but this is surely good enough for now.